Integrations/integration-Wildfire.yml

```diff
@@ -1017,7 +1054,48 @@ script:
       description: The actual score
       type: number
     description: Get a verdict regarding multiple hashes, stored in a txt file
+  - name: wildfire-get-sample
+    arguments:
+    - name: md5
+      description: MD5 hash to retrieve
+      isArray: true
+    - name: sha256
+      description: SHA256 hash to retrieve
+      isArray: true
+    outputs:
+    - contextPath: File.Size
+      description: File size
+      type: number
+    - contextPath: File.SHA1
+      description: SHA1 hash of the file
+      type: string
+    - contextPath: File.SHA256
+      description: SHA256 hash of the file
+      type: string
+    - contextPath: File.Name
+      description: The sample name
+      type: string
+    - contextPath: File.SSDeep
+      description: SSDeep hash of the file
+      type: string
+    - contextPath: File.EntryID
+      description: War-Room Entry ID of the file
+      type: string
+    - contextPath: File.Info
+      description: Basic information of the file
+      type: string
+    - contextPath: File.Type
+      description: File type e.g. "PE"
+      type: string
+    - contextPath: File MD5
+      description: MD5 hash of the file
+      type: string
+    - contextPath: File.Extension
+      description: File Extension
+      type: string
+    description: Retrieve a sample from WildFire
   - name: wildfire-upload-file-remote
+    deprecated: true
```

Comment:
should not be deprecated before the new integration is in

Reply:
we already have the wildfire-upload-file-url, so should be deprecated I believe.